src/openklant/components/klantinteracties/api/viewsets/klantcontacten.py

```diff
@@ -331,6 +332,20 @@ def perform_create(self, serializer):
             token_identifier=token_auth.identifier,
             token_application=token_auth.application,
         )
+        object_type = instance.onderwerpobjectidentificator.get("code_objecttype")
+        if object_type == "zaak":
+            process_cloudevent(
```

Comment:
I noticed that the `send_cloudevent` from `notifications-api-common` currently uses the `Content-Type: application/json` header, while the `/cloudevents` endpoint requires the `application/cloudevents+json` content-type, so if I test this locally with Open Notificaties, I get HTTP 415 error codes. Could you make sure this is changed in the library to use the `application/cloudevents+json` content-type header?